src/main/java/fr/polytech/sim/log/LoggerFactory.java

```diff
@@ -0,0 +1,7 @@
+package fr.polytech.sim.log;
+
+public class LoggerFactory {
+    public static NamedLogger createNamedLogger(String name) {
```

Comment:
Le type de retour devrai rester aussi général que possible.

```suggestion
    public static Logger createNamedLogger(String name) {
```